docs/config.md

```diff
@@ -1,457 +1,147 @@
-#
-- name: Ingest census blocks to postgres
-  command: ogr2ogr -f PostgreSQL -t_srs EPSG:4326 -nln seattle_blkgrpce10 -nlt MultiPolygon 'PG:user=postgres dbname=piecewise' /opt/piecewise_web/seattle_blkgrpce10/CENSUS10_blkgrp_WGS.shp
-
-# ------------------------------------------------------------------
-#
-# 5) Rename the folder name "seattle_example" to match your folder name. 
-#
-#    For example, change:
-#       copy: src=seattle_example/piecewise_config.json dest=/etc/piecewise/config.json
-#    to:  
-#       copy: src=baltimore_example/piecewise_config.json dest=/etc/piecewise/config.json
-
-- name: Install piecewise configuration
-  copy: src=baltimore_example/piecewise_config.json dest=/etc/piecewise/config.json
-
-# ------------------------------------------------------------------
-# The remaining Ansible commands do not need to be changed for customization of Piecewise. 
-# ------------------------------------------------------------------
-
-- name: Restart uwsgi so piecewise config is detected
-  service: name=uwsgi state=restarted
-- command: python extra_data.py chdir=/opt/piecewise
+Shapefiles are the most widely used geodata format used in the GIS community. The US Census Bureau provides downloadable shapefiles for a variety of boundaries in the US: [https://www.census.gov/geo/maps-data/data/cbf/cbf_blkgrp.html](https://www.census.gov/geo/maps-data/data/cbf/cbf_blkgrp.html). In the US, GIS people in cities or states often publish shapefiles for their communities that often amend or correct the shapefiles provided by the US Census Bureau. 
 
-```
+```TO DO: add reference to international shapefile sources```
 
-##### Edit the main Piecewise configuration file
+Once you locate the shapefile(s) you need, it's good practice to open the shapefile in [QGIS](http://www.qgis.org/en/site/) or another GIS program to confirm that it meets the requirements to use within your Piecewise application. When you open the shapefile(s) in your GIS program, **make a note of the name of its unique key field.** You'll use that later in these instructions.
 
-The main configuration file for your Piecewise deployment is ```baltimore_example/piecewise_config.json```. You will be updating some sections of this file for your deployment, most importantly the information about the aggregations you want to be applied.
+Shapefile requirements:
 
-This JSON file is arranged in sections with sub-elements. The relevant sections we will change begin with **"aggregations"** and **"filters"**. Each section below begins with a code block from the relevant section, followed by instructions on what to change, and ending with the edited section of code. In our examples, we are using the folder and file names for our Baltimore example. Your folder and file names will likely differ.
+* Must contain at least one field that serves as a unique key, for example "geoid" or "geoid10" in the case of census block groups
```

Comment:
Which files contains the unique key? The .csv has 'OBJECT_ID', but unsure if that's the correct id.